endpoints/v2/v2auth.py

```diff
@@ -255,7 +255,14 @@ def _authorize_or_downscope_request(scope_param, has_valid_auth_context):
                 # TODO: Push-to-create functionality should be configurable
                 if CreateRepositoryPermission(namespace).can() and user is not None:
                     logger.debug("Creating repository: %s/%s", namespace, reponame)
-                    found = model.repository.get_or_create_repository(namespace, reponame, user)
+                    visibility = (
+                        "private"
+                        if app.config.get("REPOSITORY_CREATION_PRIVATE", True)
```

Comment:
Can you rename the config flag to something more descriptive: `CREATE_PRIVATE_REPO_ON_PUSH` or something similar?